spalp_example/spalp_example.routing.yml

```diff
@@ -0,0 +1,8 @@
+spalp_example.get:
```

Comment:
I think it would make sense to do this as part of the main spalp module, rather than making each extending module implement the same boilerplate code

one option might be a view mode on the node, 

the URL could be as simple as `node/nid/json`

Reply:
adding it as separate endpoint for submodule would enable that module to have control over how to implement it.. example module provides boilerplate on how to do it.

Reply:
yes, but then we'll have to do the same work 20 times